cmd/delete.go

```diff
@@ -12,28 +12,29 @@ import (
 func init() {
 	root.AddCommand(deleteCmd)
 	deleteCmd.Flags().BoolP("confirm", "c", false, "Prompt to confirm all configuration options - $FAAS_CONFIRM")
-	deleteCmd.Flags().StringP("path", "p", cwd(), "Path to the project which should be deleted - $FAAS_PATH")
 	deleteCmd.Flags().StringP("namespace", "n", "", "Override namespace in which to search for Functions.  Default is to use currently active underlying platform setting - $FAAS_NAMESPACE")
 }
 
 var deleteCmd = &cobra.Command{
-	Use:               "delete <name>",
+	Use:               "delete",
```

Comment:
Why not allow to delete function by name?

Reply:
>kn service delete and kn faas delete doing exactly the same thing

Yes they practically are same thing.

Reply:
> Not sure what this mean, what conf file, what upstream?

The `.faas.yaml` file, moving to something like `.kn.yaml` in knative/client to generically describe a Knative project.